controls.py

```diff
@@ -0,0 +1,95 @@
+from testit import *
+import db
+
+
+
+def testTables():
+
+    s = db.get_session()
+
+    # Check Creation
+    A = Account(username = 'JOE',password='ultimatej03',email = 'Joeyjoe@joesph.com',account_token = '555XTZ')
+    S = Student()
+    D = Donor()
+    R = Request(requested_by = 4, amount_needed = 60.00, description = "I NEED A TOAD. TRUST ME.")
+    DM = Donation(to_request= 3, from_donor= 4,amount_given=55.9,charge_token='kEYKey', filled=False)

```

Comment:
So I am not sure exactly what you mean, this part of the code was one of the tests to see if the db worked or not.